.github/ISSUE_TEMPLATE/100-documentation.yml

```diff
@@ -0,0 +1,22 @@
+name: 📚 Documentation
+description: Report an issue related to https://docs.vllm.ai/
+title: "[Doc]: "
+labels: ["doc"]
+
+body:
+- type: textarea
+  attributes:
+    label: 📚 The doc issue
+    description: >
+      A clear and concise description of what content in https://docs.vllm.ai/ is an issue.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Suggest a potential alternative/fix
+    description: >
+      Tell us how we could improve the documentation in this regard.
+- type: markdown
+  attributes:
+    value: >
+      Thanks for contributing 🎉!
```

Comment:
nit: could you add a new line to each file?

Reply:
Fixed by [29e174e](https://github.com/vllm-project/vllm/pull/3360/commits/29e174ebd96d7934cad069afc52aa5a853fd74d1).